spec/requests/users_controller_spec.rb

```diff
@@ -2943,6 +2943,57 @@ def post_user
     end
   end
 
+  describe "#show_card" do
+    context "anon" do
+      let(:user) { Discourse.system_user }
+
+      it "returns success" do
+        get "/u/#{user.username}/card.json"
+        expect(response.status).to eq(200)
+        parsed = JSON.parse(response.body)["user"]
+
+        expect(parsed['username']).to eq(user.username)
```

Comment:
Super super super minor but why use single quotes for some attributes and double for others?